notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -61,7 +61,7 @@
     "\"git+https://github.com/huggingface/optimum-intel.git\"\\\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #68.                        history_template.format(num=round, user=item[0], assistant=item[1])
</pre><p>does this changes compatible with models where history_template does not contains round?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
no, it also works for qwen which does not includes round